source/reference/minio-mc.rst

```diff
@@ -44,6 +44,12 @@ See :ref:`minio-mc-commands` for a list of supported commands.
    is the best option for applications which trigger AGPLv3 obligations where
    open-sourcing the application is not an option.   
 
+Version
+-------
+
+The MinIO Client releases separately from the MinIO Server.
+You must use the MinIO Client version that releases at the same time (for example, on the same day) as the MinIO Server version or a later version of the MinIO Client.
```

Comment:
This text could also be the same as whatever you decide to do for troubleshooting.rst

Reply:
Replaced throughout.